### DISCUSSION TITLE
CTA component page does not have link to Hero with CTA Strip

### DISCUSSION THREAD

**Discussion by sarinapadilla** [NCIOCPL/ncids#1603] [NCIDS Launch]
From our review with the BRIDG team on 3/5, we noted that the CTA component page is missing a link to the Hero with CTA Strip variant.

**Comment by blilianyu** [comment]
On CTA component page under "When to use", we can link "Hero with CTA button" to the Hero component page.
cc: @monika-jaeger

**Comment by laurelthrash** [comment]
Discussed during 04/18 UX meeting. @adrianacastaneda to draft suggested language and page placement (and add to this discussion). We will align internally then run by Anna prior to implementing.

**Comment by adrianacastaneda** [comment]
@blilianyu @lburack I updated the ["When to use" section for the CTA strip](https://nih.sharepoint.com/:w:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20NCIDS%20Pilot_Launch/Documentation%20Site/Page%20Content%20-%20Word%20Docs/CTA%20Strip.docx?d=wb4b1f9a1b68942f690153922f59d4811&csf=1&web=1&e=Cdhms5) to reference the Hero variant and to link to the Hero page for more information. 

cc: @laurelthrash

**Reply by lburack** [reply]
Looks good, @adrianacastaneda !

**Reply by blilianyu** [reply]
Looks great to me too!! Thanks @adrianacastaneda!

**Comment by laurelthrash** [comment]
Received! Since this issue was reported by the internal team, as opposed to the federal team, during review with BRIDG and since it is not a major change, let's proceed directly with making the update. 

@andyvanavery31 @sarinapadilla are you available to please create a CR ticket for this and add to the Redesign board?

**Reply by blilianyu** [reply]
@laurelthrash @andyvanavery31 @sarinapadilla CR ticket has been created #1671
